mne/io/eyelink/tests/test_eyelink.py

```diff
@@ -192,6 +195,98 @@ def test_find_overlaps():
     assert overlap_df["eye"].iloc[0] == "both"
 
 
+def _simulate_eye_tracking_data(in_file, out_file):
+    out_file = Path(out_file)
+
+    new_samples_line = (
+        "SAMPLES\tPUPIL\tLEFT\tVEL\tRES\tHTARGET\tRATE\t1000.00"
+        "\tTRACKING\tCR\tFILTER\t2\tINPUT"
+    )
+    with out_file.open("w") as fp:
+        in_recording_block = False
+        events = []
+
+        for line in Path(in_file).read_text().splitlines():
+            if line.startswith("START"):
+                in_recording_block = True
+            if in_recording_block:
+                tokens = line.split()
+                event_type = tokens[0]
+                if event_type.isnumeric():  # samples
+                    tokens[4:4] = ["100", "20", "45", "45", "127.0"]  # vel, res, DIN
+                    tokens.extend(["1497.0", "5189.0", "512.5", "............."])
+                elif event_type in ("EFIX", "ESACC"):
+                    tokens.extend(["45", "45"])  # resolution
+                elif event_type == "SAMPLES":
+                    tokens[1] = "PUPIL"  # simulate raw coordinate data
+                    tokens[3:3] = ["VEL", "RES", "HTARGET"]
+                    tokens.append("INPUT")
+                elif event_type == "EBLINK":
+                    continue  # simulate no blink events
+                elif event_type == "END":
+                    pass
+                else:
+                    fp.write("%s\n" % line)
+                    continue
+                events.append("\t".join(tokens))
+                if event_type == "END":
+                    fp.write("\n".join(events) + "\n")
+                    events.clear()
+                    in_recording_block = False
+            else:
+                fp.write("%s\n" % line)
+
+        fp.write("%s\n" % "START\t7452389\tRIGHT\tSAMPLES\tEVENTS")
+        fp.write("%s\n" % new_samples_line)
+
+        for timestamp in np.arange(7452389, 7453390):  # simulate a second block
+            fp.write(
+                "%s\n"
+                % (
+                    f"{timestamp}\t-2434.0\t-1760.0\t840.0\t100\t20\t45\t45\t127.0\t"
+                    "...\t1497\t5189\t512.5\t............."
+                )
+            )
+
+        fp.write("%s\n" % "END\t7453390\tRIGHT\tSAMPLES\tEVENTS")
+
+
+@requires_testing_data
+@requires_pandas
+@pytest.mark.parametrize("fname", [(fname_href)])
+def test_multi_block_misc_channels(fname):
+    """Test an eyelink file with multiple blocks and additional misc channels."""
+    out_file = Path("./tmp_eyelink.asc")
```

Comment:
```suggestion
def test_multi_block_misc_channels(fname, tmp_path):
    """Test an eyelink file with multiple blocks and additional misc channels."""
    out_file = tmp_path / "tmp_eyelink.asc"
```
then at the end you don't have to `.unlink()` or anything.